turbinia/api/cli/README.md

```diff
@@ -1 +1,164 @@
+
+Support for multiple Turbinia environments is possible in the configuration file. In the example below, there are two Turbinia environments, ```default``` and ```development```.
+
+```
+{
+	"default": {
+		"description": "This file is used by turbinia-client to determine the location of the API server and if authentication will be used. These options should match your Turbinia deployment.",
+		"comments": "By default, the credentials and client secrets files are located in the user's home directory.",
+		"API_SERVER_ADDRESS": "http://localhost",
+		"API_SERVER_PORT": 8000,
+		"API_AUTHENTICATION_ENABLED": false,
+		"CLIENT_SECRETS_FILENAME": ".client_secrets.json",
+		"CREDENTIALS_FILENAME": ".credentials_default.json"
+	},
+	"development": {
+		"description": "Development environment.",
+		"API_SERVER_ADDRESS": "http://localhost",
+		"API_SERVER_PORT": 8001,
+		"API_AUTHENTICATION_ENABLED": false,
+		"CLIENT_SECRETS_FILENAME": ".client_secrets_dev.json",
+		"CREDENTIALS_FILENAME": ".credentials_dev.json"
+	}
+}
+```
+A specific environment can be accessed by setting the ```TURBINIA_CONFIG_INSTANCE``` environment variable to a valid environment name from the configuration file or alternatively, via the ```-c``` argument. (e.g. ```turbinia-client -c development config list```)
+
+### Usage
+You can view the help menu with the ```-h``` argument
+
+```
+$ turbinia-client -h
+
+Usage: turbinia-client [OPTIONS] COMMAND [ARGS]...
+
+  Turbinia API command-line tool (turbinia-client).
+
+                          ***    ***
+                           *          *
+                      ***             ******
+                     *                      *
+                     **      *   *  **     ,*
+                       *******  * ********
+                              *  * *
+                              *  * *
+                              %%%%%%
+                              %%%%%%
+                     %%%%%%%%%%%%%%%       %%%%%%
+               %%%%%%%%%%%%%%%%%%%%%      %%%%%%%
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%  ** *******
+  %%                                                   %%  ***************
+  %%                                (%%%%%%%%%%%%%%%%%%%  *****  **
+    %%%%%        %%%%%%%%%%%%%%%
+    %%%%%%%%%%                     %%          **             ***
+       %%%                         %%  %%             %%%           %%%%,
+       %%%      %%%   %%%   %%%%%  %%%   %%%   %%  %%%   %%%  %%%       (%%
+       %%%      %%%   %%%  %%%     %%     %%/  %%  %%%   %%%  %%%  %%%%%%%%
+       %%%      %%%   %%%  %%%     %%%   %%%   %%  %%%   %%%  %%% %%%   %%%
+       %%%        %%%%%    %%%       %%%%%     %%  %%%    %%  %%%   %%%%%
+
+  This command-line tool interacts with Turbinia's API server.
+
+  You can specify the API server location in ~/.turbinia_api_config.json
+
+Options:
+  -c, --config_instance TEXT  A Turbinia instance configuration name.
+                              [default: (dynamic)]
```

Comment:
That is because in the code the default value is generated by a lambda function which returns either a default string, or the value in the environment variable (if it is set), hence in the help message it shows "dynamic".